games-emulation/handy_libretro/handy_libretro-0.95_20180219.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A port of Handy, an Atari Lynx emulator for the libretro API"
+DESCRIPTION="Handy is an emulator of the Atari Lynx handheld. It requires the \
+Lynx BIOS to boot and start games."
+HOMEPAGE="http://handy-dev.blogspot.com/"
```

Comment:
It's 404 for me.

Reply:
Same thing for me.
(BTW, the site first issues a (geographic/country-based) 302 redirect to handy-dev.blogspot.\<CountryTLD\> depending on the country of the client IP, e.g. `.fr` or `.co.uk`.
So maybe the redirect URI only works for users coming from countries where Free speech is protected by the constitution :smile:
(That's not the case in the first example above :cry:)

Reply:
Shall we use this?
```
HOMEPAGE="https://github.com/libretro/libretro-handy
	http://handy-dev.blogspot.com/"
```
@kwyxz, I can push a tiny commit with this change if you wish.

Reply:
@kwyxz & @alaviss: what's your favorite `HOMEPAGE`?
* A)
```
HOMEPAGE="https://github.com/libretro/libretro-handy
	http://handy-dev.blogspot.com/"
```
* B)
```
HOMEPAGE="https://github.com/libretro/libretro-handy"
```
* C)
```
HOMEPAGE="http://handy-dev.blogspot.com/"
```
* D)
```
HOMEPAGE="http://handy-dev.blogspot.com/
	https://github.com/libretro/libretro-handy"
```

I'd use A, but I'm OK with B and D too. We should probably avoid C, which is what we have right now in the PR.

Shall I push an update with one of these changes? (Let me know which you prefer :)

Reply:
Looks like the blogspot website is mostly dead
But this is alive http://handy.sourceforge.net/

Reply:
That's not the site for the libretro port.
What about:
```
HOMEPAGE="https://github.com/libretro/libretro-handy
	http://handy.sourceforge.net/"
```
or
```
HOMEPAGE="http://handy.sourceforge.net/
	https://github.com/libretro/libretro-handy"
```
That said, I'm OK with `HOMEPAGE="http://handy.sourceforge.net/"`, so I can merge the PR as it is now, if you wish.

Reply:
Homepages so far have never been the sites for the libretro port but always of the original program that is being ported to libretro.

Reply:
That is, because libretro ports are usually just github repos. I believe it is more respectful to the original author to provide the original program homepage.